org.free_astro.siril.json

```
@@ -192,7 +192,8 @@
             ]
         },
         {
-            "name": "libraw",
+            "name": "
+            ",
```

```suggestion
            "name": "libraw",
```

This explain the build failure